src/DynamoRevit/DynamoRevit.cs

```diff
@@ -183,10 +194,21 @@ public Result ExecuteCommand(DynamoRevitCommandData commandData)
                     InitializeCoreView().Show();
                 }
 
+                //foreach preloaded exception send a notification to the Dynamo Logger
+                //these are messages we want the user to notice.
+                preLoadExceptions.ForEach(x => revitDynamoModel.Logger.LogNotification
+                (revitDynamoModel.GetType().ToString(),
+                x.GetType().ToString(),
+                DynamoApplications.Properties.Resources.MismatchedAssemblyVersionShortMessage,
+                x.Message));
+
                 TryOpenAndExecuteWorkspace(extCommandData);
 
                 // Disable the Dynamo button to prevent a re-run
                 DynamoRevitApp.DynamoButtonEnabled = false;
+
+                //unsubscribe to the assembly load
+                AppDomain.CurrentDomain.AssemblyLoad -= AssemblyLoad;
```

Comment:
we only subscribe to this event during construction and load of Dynamo Model and View.